### PR TITLE
Feature/bphh 733/archive requirement block

### PR DESCRIPTION
**Pull request by taw27**
## Description
Archive requirement block

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📦 Chore (Release)
- [ ] ✅ Test

## Related Tickets & Documents
 https://hous-bssb.atlassian.net/browse/BPHH-733

## Steps to QA

https://github.com/user-attachments/assets/c4e6680c-e167-4045-b934-124541c7b50e

### CODE REVIEW

File: app/frontend/hooks/resources/use-requirement-template.ts [hunk 1]
```diff
@@ -20,17 +20,15 @@ export const useRequirementTemplate = () => {
       return
     }
 
-    if (!requirementTemplate || !requirementTemplate.isFullyLoaded) {
```
Comment (bkmaibach) [inline app/frontend/hooks/resources/use-requirement-template.ts:23]: Why was this removed?

**Review by bkmaibach — APPROVED** [review]

File: app/frontend/hooks/resources/use-requirement-template.ts (hunk 1 above)
Comment (taw27) [reply app/frontend/hooks/resources/use-requirement-template.ts:23]: I want to get the latest data on every page visit. Right now if the requirement template is fully loaded once, it won't do a refetch. This can lead to stale data
